### DISCUSSION TITLE
【灌水】你们这是绳么网啊！！！

### DISCUSSION THREAD

**Discussion by 127kzzz** [share121/inter-knot#366] [其他]
![1402904860_1660577164797](https://github.com/user-attachments/assets/1fc22202-de89-4aa7-be74-7fade63a9829)

你们这个是什么网啊！！！
你们这是害人不浅啊你们这个网！！！！
真的太过分了你们搞这个网干什么！我儿子每一科的成绩都不过那个平均分呐！！！！！
他现在初二。你叫我儿子怎么办啊┴┴︵╰（‵□′）╯︵┴┴
他还不到高中啊好不好！！！！！
你们这是绳么网啊，你们害死我儿子了Σ(｀д′*ノ)ノ
谁是群主快点出来你们群主，再不来我去报警了啊————！
我跟你们说你们这一帮人啊！一天到晚啊搞这些什么空洞啊委托啊会害死你们哒！！！
你们没有前途我跟你说你们这么多个人好好学习不好吗？一天到晚在上网有什么意思！有什么意思啊！！！
麻烦你们重视一下你们生活的目标好不好有一点学习目标好不好！！！
一天到晚上绳网是不是人啊！！！，你们一天到晚上绳网网不好好学习会没前途的！空洞害死人(;´༎ຶД༎ຶ`)！！！

**Comment by mizukanain** [comment]
这位阿姨，请不要激动

**Comment by wugan-bot156** [comment]
哈哈哈哈，经典，太经典了

**Comment by whoisfucker** [comment]
错了，错了，马上改正

**Comment by c1iied** [comment]
这孩子现在估计都快大二了

**Comment by ShangHuaWenYuan** [comment]
[一❤个❤人❤在❤家❤看❤伊❤埃❤斯❤热❤舞](https://www.bilibili.com/video/BV1HT421k7Rp/?spm_id_from=333.337.search-card.all.click&vd_source=9dbf0038cac01a66dbf38722f8cdaf31)
